package.json

```diff
@@ -31,13 +32,13 @@
     "lodash.flatten": "^4.4.0",
     "lru": "^3.1.0",
     "moment-timezone": "0.5.11",
-    "react": "0.14.8 || ^15.5.0 || ^16.0.0",
     "xgettext-js": "1.0.0"
   },
   "devDependencies": {
     "chai": "^3.5.0",
     "enzyme": "2.9.1",
     "mocha": "^2.4.5",
+    "react": "0.14.8 || ^15.5.0 || ^16.0.0",
```

Comment:
Hmm, why is `react` a `devDependency` now?

Reply:
it used in tests but not in the actual code

Reply:
Sure about that? https://github.com/Automattic/i18n-calypso/pull/44/files?diff=unified#diff-5f10a981b0463af0a44369280566e2e3R1

Reply:
> it used in tests but not in the shipped code

FAKE NEWS ALERT